### ISSUE TITLE
Cuted slide and lack of responsivness in center mode

### ISSUE THREAD

**Issue by draszka** [kenwheeler/slick#3151]
Hello,

I have an issue with cutted slide in my slider and broken responsivness. Slider is set as below with use of center mode. All calculation seems to work properly (picutres), but inspite of it slider breaks. 

https://imgur.com/a/Z2Rni
https://imgur.com/a/Coqt8
https://imgur.com/a/l9qtM

`{
            centerMode: true,
            centerPadding: '18%',
            slidesToShow: 1,
            slide: 'article',
            appendArrows: $('.video.slider nav'),
            nextArrow: '<button type="button" class="next"><span class="icon icon-video-next"></span></button>',
            prevArrow: '<button type="button" class="previous"><span class="icon icon-video-prev"></span></button>',
            focusOnSelect: true,
            cssEase: 'cubic-bezier(.25,.46,.45,.94)',
            speed: 1000
},`

One of the slides has video put asynchronously in one slide, so I was trying, with use of setPosition recalculate slick math, but this still not work. 

set new width:
`$('.video.slider .track').find(".slick-slide").width("auto");`
and after that setPosition:
`$('.video.slider .track').slick("setOption", '', '', true);`

Do you have any idea what is the problem and how can I fix it?

Best regards!

**Comment by draszka** [comment]
No one had similar problem? Please give me any direction, I got stuck...:(

**Comment by divy3993** [comment]
I have wasted so much time, I thought I was doing something wrong, but found similar issues open here on git...

Faced similar problem so I have customized the slick 'centerMode' value to 'true' or 'false' based on screen width. It is as below:

`var centerModeValue = ($(window).outerWidth() > 639) ? true : false;`

So this variable stores 'true' if screen width is above 639px where I want centerMode to be activated, and below sscreen width 639px it will be deactivated with the 'false' flag from same variable value.

```
$('.slider-wrapper').slick({
centerMode: centerModeValue,
});
```

Hope this helps, someone who is really in need to make it work.

Thanks!
